.travis.yml

```diff
@@ -40,9 +40,19 @@ install:
         luarocks install busted
         luarocks install cluacov
         luarocks make
+    else
+        TARANTOOL_VERSION=1.10
```

Comment:
I would use `TARANTOOL_VERSION=${TARANTOOL_VERSION:-1.10}`: after that we can add more jobs on different tarantool versions or change a tarantool version using Travis-CI environment variables.

Reply:
I think this should be moved to build matrix then. Now it runs on the oldest supported release, which has fewer features than modern ones.

Reply:
I don't mind.